src/test/functional/features/sole-conditional-order.feature

```diff
@@ -43,3 +43,14 @@ Feature: Sole conditional order
     And I click "I believe that the facts stated in this application are true"
     And I click "Continue"
     Then the page URL should be "/hub-page"
+    Given I set the case state to "AwaitingPronouncement"
+    And a case worker updates court case hearing
+    When I click "Sign out"
+    And I login with applicant "1"
+    Then the page URL should be "/hub-page"
+    And the page should include "The hearing will take place at Birmingham Civil and Family Justice Centre on 29 September 2013 at 3:30PM."
+    When I click "Sign out"
+    And I login with applicant "2"
+    When I go to "/"
+    Then the page URL should be "/respondent/hub-page"
+    And the page should include "The hearing will take place at Birmingham Civil and Family Justice Centre on 29 September 2013 at 3:30PM."
```

Comment:
Is there a reason this is added onto 'Applicant 1 applies for condition order' rather than in the sole-hub-page.feature file?

Reply:
I felt like as its still part of condition order process, it should belong here but i guess as this is specifically looking at the hub page, i can move it there

Reply:
done